examples/src/main/resources/quijote.txt

```diff
@@ -0,0 +1,73 @@
+El ingenioso hidalgo don Quijote de la Mancha
```

Comment:
Silly but not so silly question. Is this a copyright free text?

Reply:
Yup, as long it is not a translation its good to go :D